common/history/global/imperia_global.txt

```diff
@@ -18,6 +18,10 @@
             name = political_discord
             value = 0
         }
+		set_variable = {
```

Comment:
Mixed indentation

Reply:
![image](https://github.com/Imperia-Development-Victoria-3/Victoria-3-Dev/assets/44327039/8fe83b59-b1c4-41d6-bc2b-78a3e941e33d)
I'll give it an autoformat anyways :s